tda/matrix.py

```diff
@@ -0,0 +1,289 @@
+from copy import copy, deepcopy
+from itertools import chain
+
+
+class Matrix(object):
+    """
+
+    Representation of matrices.
+
+    """
+
+    def __init__(self, entries):
+        """
+
+        Parameters:
+        -----------
+        entries: list
+            Two dimensional list of entries.
+
+        Notes:
+        ------
+        Entries have to be integers.
+
+        """
+
+        self.entries = entries
+        self.num_rows = len(entries)
+        self.num_cols = len(entries[0])
+
+    def get_row(self, row_index):
+        """
+
+        Get a row from matrix.
+
+        Parameters:
+        -----------
+        row_index: int
+            Index of a row.
+
+        Output:
+        -------
+        row: list
+            A row.
+
+        """
+
+        return self.entries[row_index]
+
+    def get_col(self, col_index):
+        """
+
+        Get a column from matrix.
+
+        Parameters:
+        -----------
+        col_index: int
+            Index of a column.
+
+        Output:
+        -------
+        col: list
+            A column.
+
+        """
+
+        col = []
+        for row in self.entries:
+            col.append(row[col_index])
+        return col
+
+    def change_row(self, row_index, values):
```

Comment:
Verify that `0 <= row_index < self.num_rows`.
Verify that `len(values) == len(self.entries[0])`.